### DISCUSSION TITLE
publish original analog value after triggering with ANALOG#A0div10

### DISCUSSION THREAD

**Discussion by taHC81** [arendst/Tasmota#11453] [Q&A] [answered]
Hello everyone, tried almost everything, but no success. I need to trigger action based on analog pin value (photoresistor), but not so often - so I've used ANALOG#A0div10.
`Rule3 ON analog#a0div10 DO publish tele/%topic%/SENSOR {"ANALOG":{"A0":%value%}} ENDON`
My issue is that 10% value change on A0 triggers the publish command where %value% = 0-100, but sensor tele message on configured interval (def. TelePeriod 300) reports value 0-1023. Is it somehow possible to publish original (non-DIV10) value via this rule? Any comments welcomed!

**Comment by sfromis** [comment, marked as answer]
Using `ANALOG#A0div10` makes the unscaled value unavailable to the same rule. You might be able to get it using another rule triggering on `ANALOG#A0`, putting it into a var. I'd try putting that second rule first, and then see if it would work to reference that var in a publish in the `ANALOG#A0div10` rule.

**Reply by taHC81** [reply]
Thanks, pointed me to a Rules article, and this one was the right for me - https://tasmota.github.io/docs/Rules/#transmit-sensor-value-only-when-a-delta-is-reached

Here's the final rule:
`Rule3 ON ANALOG#A0 DO Var1 %value% ENDON ON analog#a0div10 DO publish tele/%topic%/SENSOR {"ANALOG":{"A0":%Var1%}} ENDON`
